content/concepts/did-ddo.md

```diff
@@ -34,98 +43,120 @@ In Ocean, a DID is a string that looks like:
 ```text
 did:op:0ebed8226ada17fde24b6bf2b95d27f8f05fcce09139ff5cec31f6d81a7cd2ea
 ```
+where "0ebed8226ada17fde24b6bf2b95d27f8f05fcce09139ff5cec31f6d81a7cd2ea" is the address of the ERC721 contract that represents that asset.
 
 It follows [the generic DID scheme](https://w3c-ccg.github.io/did-spec/#the-generic-did-scheme).
 
-The part after `did:op:` is the asset's on-chain Ethereum address (minus the "0x"). One can be computed from the other; therefore there is a 1:1 mapping between did and Ethereum address.
 
 ## DDO Attributes
 
-![DDO Content](images/ddo-content.png)
 
 A DDO has these standard attributes:
 
-- `@context`
-- `id`
-- `created`
-- `updated`
-- `publicKey`
-- `authentication`
-- `proof`
-- `verifiableCredential`
+- `@context`   = array, contexts used for validation
+- `id`  = string, address of ERC721 contract
+- `created` = updated by aquarius, contains the date of publishing (block.timestamp)
+- `updated`  = updated by aquarius, contains the date of the update (block.timestamp)
+- `proof` = proof of ownership 
```

Comment:
what is the point of this? and there is no explanation of it here. It currently looks like
```
  "proof": {
    "created": "2021-06-06T19:30:28Z",
    "creator": "0xDF1c35c3d5178e9d98043b35a6737Bd861c191c9",
    "type": "AddressHash",
    "signatureValue": "0x0e870c8dde468de839b55456604e3d81e0e4e5327ea2d9a5188f3b0f657d1abf"
  },
```
It has mostly redundant values and other that have no purpose.

Reply:
made it optional, but we need it in order to comply with the standard
see https://w3c-ccg.github.io/did-spec/#the-generic-did-scheme


verification method
A set of parameters that can be used together with a process to independently verify a proof. For example, a cryptographic public key can be used as a verification method with respect to a digital signature; in such usage, it verifies that the signer possessed the associated cryptographic private key.

"Verification" and "proof" in this definition are intended to apply broadly. For example, a cryptographic public key might be used during Diffie-Hellman key exchange to negotiate a shared symmetric key for encryption. This guarantees the integrity of the key agreement process. It is thus another type of verification method, even though descriptions of the process might not use the words "verification" or "proof."

Reply:
The latest version of the DID spec does not include the "proof" property.  See for details: https://www.w3.org/TR/did-core/#did-document-properties

It was removed after the First Public Working Draft was released. 

Quoted from https://www.w3.org/TR/did-core/#revision-history
>Changes since the First Public Working Draft include:
>...
>* The removal of embedded proofs in the DID Document.

Reply:
I removed all mentions of `ddo.proof`